nerfstudio/viewer/app/src/modules/Scene/Scene.jsx

```diff
@@ -107,20 +111,20 @@ export function get_scene_tree() {
       camera_controls.setTarget(
         curPos.x + diff.x,
         curPos.y + diff.y,
-        curPos.z + diff.z,
+        curPos.z + diff.z,true
```

Comment:
Write out argument name, hard to tell what this is referring to. (here and below)